msteams-platform/whats-new.md

```diff
@@ -211,6 +212,7 @@ Developer preview is a public program that provides early access to unreleased T
 
 | **Date** | **Update** | **Find here** |
 | -------- | --------- | ------------------ |
+|12/28/2021| Introduced step-by-step guide to do Teams file upload bot | Build bots > Send and receive files > [step-by-step guide to grant Teams file upload bot](sbs-file-handling-in-bot.yml) |
```

Comment:
Remove from Dev preview and add under GA

Reply:
changes made

Reply:
changes made